apps/oxfmt/src/cli/format.rs

```diff
@@ -260,11 +263,16 @@ fn load_config_path(cwd: &Path, config_path: Option<&Path>) -> Option<PathBuf> {
     })
 }
 
+/// Returns: (ignore_patterns, is_sort_package_json)
+type OxfmtOnlyOptions = (Vec<String>, bool);
```

Comment:
Better to have a struct? We may have more options in the future. Passing a single `bool` around is not that nice as well.